src/qix.js

```diff
@@ -20,13 +20,14 @@ class Qix {
     const {
       url,
       createSocket,
-      delta,
+      protocol,
       Promise,
       responseInterceptors,
       JSONPatch,
       definition,
     } = config;
-    const apis = new ApiCache({ Promise, definition });
+    const delta = protocol.delta;
```

Comment:
I imagined that we would pass `protocol` into `new Session()` and let than do `Object.assign()` on the request, that way we allow the end-user to pass in (potentially undocumented) request parameters to QIX Engine without us having to modify any source code. That would be more in line to how enigma.js works with schemas.